cluster/images/etcd-version-monitor/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Please remove this licence. This is part of Kubernetes and should be under its licence.